### ISSUE TITLE
Get published in Nextcloud app store

### ISSUE THREAD

**Issue by 37Rb** [37Rb/nextcloud-hledger#9]
Do all the things required and get in there.

**Comment by Stochastic13** [comment]
Hi! Is there are development on this? Or any timeline to expect an app-store ready package? In the meanwhile, what is the simplest way to add this app into a custom nextcloud instance?

**Comment by thecount2a** [comment]
Honestly, @37Rb and I did a bunch of work on this a while back but at this point we've stopped maintaining this. I am personally working on a new syncing Android+Windows/Linux Desktop app that will be completely outside of the nextcloud app ecosystem (although it may support nextcloud as a storage platform) but this app is still very much in-progress and not yet ready for production. I do plan to announce it to the hledger mailing list when it's in a beta state so if you're interested in it, make sure you're subscribed to that mailing list.

**Comment by 37Rb** [comment]
Correct. I still think it's a good idea but I no longer have time available to work on it. Here's an email thread in the google group about it.

https://groups.google.com/g/hledger/c/oyHfEf4UwIM/m/ihl7B2RsAQAJ

**Comment by Stochastic13** [comment]
@37Rb @thecount2a Thanks! I totally understand. Unfortunately, I do not have php/vue experience which might be needed to contribute to this. I assume there is no alternative that any of you might know of? I did check and found nothing. Currently using local accounting software and periodically saving it on NC.

**Comment by 37Rb** [comment]
I'm still using hledger, mostly the console app hledger-ui, in a git repo that I keep in a folder synced with nextcloud. So I can view the journal file in nextcloud if I want but usually I work with it locally.
